### ISSUE TITLE
`lang: zh` doesn't translate Appendix to `附录`  when output is html

### ISSUE THREAD

**Issue by XiangyunHuang** [quarto-dev/quarto-cli#4578]
## Step 1.  Create book project

```bash
quarto create-project mybook --type book
```

## Step 2.  Set `lang: zh` and add `appendices` in `_quarto.yml`

```yml
project:
  type: book

lang: zh

book:
  title: "mybook"
  author: "Jane Doe"
  date: "3/1/2023"
  chapters:
    - index.qmd
    - intro.qmd
    - references.qmd
  appendices:
    - summary.qmd

bibliography: references.bib

format:
  html:
    theme: cosmo
  pdf:
    documentclass: scrreprt
```

## Step 3. Render Book

<img width="839" alt="appdix" src="https://user-images.githubusercontent.com/12031874/222173416-7a78085b-1171-47ab-87c6-a883fd6d9e04.png">

## Session Info

I have tested in Quarto 1.2.269 and Quarto 1.2.335  and Quarto 1.3.235

RStudio 2022.12.0+353 "Elsbeth Geranium" Release (7d165dcfc1b6d300eb247738db2c7076234f6ef0, 2022-12-03) for macOS
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) RStudio/2022.12.0+353 Chrome/102.0.5005.167 Electron/19.1.3 Safari/537.36

### Checklist

- [X] Please include a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.
- [X] Please [format your issue](https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us) so it is easier for us to read the bug report.
- [X] Please document the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?
- [X] Please document the operating system you're running. If on Linux, please provide the specific distribution.

**Comment by XiangyunHuang** [comment]
I also come across some warnings when using `Quarto Render`

```
Quarto v1.3.235
[1/4] index.qmd
[WARNING] Could not load translations for zh
  data file translations/zh.yaml not found
[WARNING] The term Abstract has no translation defined.
[2/4] intro.qmd
[WARNING] Could not load translations for zh
  data file translations/zh.yaml not found
[WARNING] The term Abstract has no translation defined.
[3/4] references.qmd
[WARNING] Could not load translations for zh
  data file translations/zh.yaml not found
[WARNING] The term Abstract has no translation defined.
[4/4] summary.qmd
[WARNING] Could not load translations for zh
  data file translations/zh.yaml not found
[WARNING] The term Abstract has no translation defined.

pandoc 
  to: latex
  output-file: index.tex
  standalone: true
  toc: true
  number-sections: true
  top-level-division: chapter
  pdf-engine: xelatex
  variables:
    graphics: true
    tables: true
  default-image-extension: pdf
  
metadata
  crossref:
    chapters: true
  documentclass: scrreprt
  papersize: letter
  classoption:
    - DIV=11
    - numbers=noendperiod
  header-includes:
    - '\KOMAoption{captions}{tableheading}'
  block-headings: true
  lang: zh
  bibliography:
    - references.bib
  title: mybook
  author: Jane Doe
  date: 3/1/2023
  
running xelatex - 1
  This is XeTeX, Version 3.141592653-2.6-0.999994 (TeX Live 2022) (preloaded format=xelatex)
   restricted \write18 enabled.
  entering extended mode
  
running xelatex - 2
  This is XeTeX, Version 3.141592653-2.6-0.999994 (TeX Live 2022) (preloaded format=xelatex)
   restricted \write18 enabled.
  entering extended mode
  

Output created: _book/index.html
```

**Comment by dragonstyle** [comment]
Thanks for reporting - this should be resolved in the next pre-release build (f68ae776f91edc6e0a5fdf95f64791eceb4b1d25).
